plugin/metric/collector/instance.go

```diff
@@ -0,0 +1,51 @@
+package collector
+
+import (
+	"context"
+
+	"github.com/bytebase/bytebase/api"
+	"github.com/bytebase/bytebase/store"
+	"go.uber.org/zap"
+)
+
+// instanceCollector is the metric data collector for instance.
+type instanceCollector struct {
+	l     *zap.Logger
+	store *store.Store
+}
+
+// instanceEventName is the MetricEventName for instance
+var instanceEventName MetricEventName = "bb.instance"
```

Comment:
Let's aggregate the metricName into a single place, so it's easier to see all metrics we collect and make sure it forms a consistent naming pattern.